ui/app/AppLayouts/Communities/panels/MembersTabPanel.qml

```diff
@@ -267,30 +267,32 @@ Item {
             icon.width: 40
             icon.height: 40
 
-            onClicked: {
-                if (mouse.button === Qt.RightButton) {
-                    const profileType = Utils.getProfileType(model.isCurrentUser, false, model.isBlocked)
-                    const contactType = Utils.getContactType(model.contactRequest, model.isContact)
-
-                    const params = {
-                        profileType, contactType,
-                        pubKey: model.pubKey,
-                        compressedPubKey: model.compressedPubKey,
-                        emojiHash: root.utilsStore.getEmojiHash(model.pubKey),
-                        colorHash: model.colorHash,
-                        colorId: model.colorId,
-                        displayName: memberItem.title || model.displayName,
-                        userIcon: model.icon,
-                        trustStatus: model.trustStatus,
-                        onlineStatus: model.onlineStatus,
-                        ensVerified: model.isEnsVerified,
-                        hasLocalNickname: !!model.localNickname
-                    }
+            onClicked: function () {
+                Global.openProfilePopup(model.pubKey)
+            }
 
-                    memberContextMenuComponent.createObject(root, params).popup(this)
-                } else if (mouse.button === Qt.LeftButton) {
-                    Global.openProfilePopup(model.pubKey)
+            onRightClicked: function () {
+                // FIXME: onClicked is called at the same time as onRightClicked (could be a QT5 only issue)
```

Comment:
This is a fix because the clicks didn't work anymore, but now the right-click is also broken. 

It probably got introduced during the QT6 migration cc @micieslak @caybro

Reply:
Were you testing on Qt5 or Qt6? Anyway, feel free to report a separate issue for this component, I'll have a look

Reply:
Qt5